lib/zclient.c

```diff
@@ -1706,6 +1719,17 @@ static void zclient_interface_down(struct zclient *zclient, vrf_id_t vrf_id)
 	if_down_via_zapi(ifp);
 }
 
+static void zclient_handle_error(ZAPI_CALLBACK_ARGS)
+{
+	enum zebra_error_types error;
+	struct stream *s = zclient->ibuf;
+
+	zapi_error_decode(s, &error);
```

Comment:
weird - I don't anything that changes the SA warning: can't `error` still end up uninitialized?

Reply:
God I had a hell of time with this SA. I could get it to pass with a memset in the function on my local clang9 analyzer but not the CI's clang8 one.

The clang8 required me to change the API to STREAM_GET and then it passed.

I will add back in the memset as well to be safe.